portal-web/test/functional/com/liferay/portalweb/paths/FrontendSPAInfrastructure.path

```diff
@@ -0,0 +1,24 @@
+<html>
+<head>
+<title>FrontendSPAInfrastructure</title>
+</head>
+<body>
+<table cellpadding="1" cellspacing="1" border="1">
+<thead>
+<tr><td rowspan="1" colspan="3">Pawel</td></tr>
+</thead>
+<tbody>
+<tr>
+	<td>Disable_SPA</td>
+	<td>//label[(.='Disable SPA in Internet Explorer')]/input</td>
+	<td></td>
```

Comment:
can move to Checkbox.function, rename to ANY_CHECKBOX_SPECIFIC. And make "Disable SPA in Internet Explorer" generic variable for anyone else to use.